sys-block/gpart/gpart-0.3.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="A small tool which tries to guess what partitions are on a hard disk"
+DESCRIPTION="Gpart is a small tool which tries to guess what partitions are \
+on a PC type, MBR-partitioned hard disk in case the primary partition table \
+was damaged."
+HOMEPAGE="https://github.com/baruch/gpart"
+COPYRIGHT="1999-2001 Michail Brzitwa"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/baruch/gpart/archive/0.3.zip"
+CHECKSUM_SHA256="6e2e1727af22615853d78958745230dd3a554b8e87c3212892ca7bd7117b1635"
+PATCHES="gpart-$portVersion.patchset"
+
+ARCHITECTURES="x86 x86_64"
```

Comment:
x86 is not supported anymore, so you should mark it as untested (with a "?x86").
```diff
-ARCHITECTURES="x86 x86_64"
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
```
In case it does not build on x86_gcc2 primary arch, it is possible to add x86 as a secondary arch.